src/component/view/bootstrap/entity-form.html

```diff
@@ -0,0 +1,19 @@
+<template>
+  <aurelia-form behavior.bind="behavior">
+    <div>
+      <slot>
+        <slot name="before"></slot>
+
+        <form-group repeat.for="element of elements"
+          if.bind="isVisible(element.field)"
+          label.bind="element.field"
+          element.bind="element.element"
+          value.bind="entity[element.field]"
+          behavior.bind="behavior"
```

Comment:
It doesn't pass the behavior through because of the `div` wrapper (needed for the slots)